adal/src/main/java/com/microsoft/aad/adal/Logger.java

```diff
@@ -305,12 +305,11 @@ private static String getCodeName(ADALError code) {
         return "";
     }
 
-    private static String GetUTCdatetimeAsString() {
+    private static String GetUTCDateTimeAsString() {
```

Comment:
not sure what the question is. Android Studio pointed out our camel casing was wrong

---
In reply to: [65642931](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/652#discussion_r65642931) [](ancestors = 65642931)